README.md

```diff
@@ -116,9 +116,9 @@ The scripts consider some events to be “changes”, “deploys”, and “inci
 
 1.  Update the nightly scripts in BigQuery for the following tables:
 
-    *   `four\_keys.changes`
-    *   `four\_keys.deployments`
-    *   `four\_keys.incidents`
+    *   `four_keys.changes`
+    *   `four_keys.deployments`
+    *   `four_keys.incidents`
 
     To update the scripts, we recommend that you update the `sql` files in the `queries` folder, rather than in the BigQuery UI.
```

Comment:
If the queries are edited in source, how do those changes get propagated back up to BigQuery?

Reply:
That's the next line!  You have to run the schedule.py script.

Reply:
I told you I don't read. :)